relay/solidity/migrations/btc-networks.js

```diff
@@ -13,9 +13,9 @@ module.exports = {
     },
 
     testnet: {
-        genesis: '0x0000c0205d1103efc13e6647977e3d65f253c3e762451e9ca9b920517d000000000000008442a07bcde3292a888277ea6337ba5bbdfa808ae01535846f19d843144c8f60478bb15e7b41011a88be5d36',
-        height: 1723030,
-        epochStart: '0xe2657f702faa9470815005305c45b4be2271c22ade1348e6fe00000000000000',
+        genesis: '0x0000453faa4dcc559d47ed24be33b5539638b797e2dbd7527e61d8c13f00000000000000c4cb592bba9489c40ae5b12efa0f39e4d98dd6d58b794b65f6c50e450ee9957081b96160c0ff3f19015ae4ae',
```

Comment:
I'm wondering if it would be worth replacing the hardcoded values with some function that will calculate them on the contract deployment. But this is something for the future.

Reply:
Yeah, we have such a script in `local-setup`: https://github.com/keep-network/local-setup/blob/master/relay-genesis/relay-genesis.js . We need to think how to use it here in the future.